cmd/run.go

```diff
@@ -125,12 +127,23 @@ func runRun(ctx context.Context, ro options.RunOptions, args []string, signers .
 		roHashes = append(roHashes, cryptoutil.DigestValue{Hash: hash, GitOID: false})
 	}
 
+	for _, dirHashGlobItem := range ro.DirHashGlobs {
+		if strings.Contains(dirHashGlobItem, "**") {
+			return fmt.Errorf("failed to parse dirhash-glob: can not contain deep (**) glob matches.")
+		}
```

Comment:
I might need to rething this. I thought about it while developing, but I think it could also be removed and allow full globs. I don't recall my reasoning at the time anymore.